xtask/src/main.rs

```diff
@@ -0,0 +1,44 @@
+use std::path::PathBuf;
+use std::process;
+use xtask_wasm::{anyhow::Result, clap};
+
+#[derive(clap::Parser)]
+enum Cli {
+    Dist(xtask_wasm::Dist),
+    Watch(xtask_wasm::Watch),
+    Start(xtask_wasm::DevServer),
+}
+
+fn main() -> Result<()> {
+    let cli: Cli = clap::Parser::parse();
+
+    match cli {
+        Cli::Dist(args) => {
+            let dist = args.static_dir_path("static").run("{{project-name}}")?;
```

Comment:
This was meant to be used this way:

```suggestion
            let DistResult { dist_dir, .. } = args.static_dir_path("static").run("{{project-name}}")?;
```